src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -98,6 +99,21 @@ class NOCChain:
     adminSubject: int
 
 
+@dataclass
+class ICDRegistrationParameters:
+    symmetricKey: typing.Optional[bytes]
+    checkInNodeId: typing.Optional[int]
+    monitoredSubject: typing.Optional[int]
+    stayActiveMs: typing.Optional[int]
+
+    class CStruct(Structure):
+        _fields_ = [('symmetricKey', c_char_p), ('symmetricKeyLength', c_size_t), ('checkInNodeId',
+                                                                                   c_uint64), ('monitoredSubject', c_uint64), ('stayActiveMsec', c_uint32)]
+
+    def to_c(self):
+        return ICDRegistrationParameters.CStruct(self.symmetricKey, len(self.symmetricKey), self.checkInNodeId, self.monitoredSubject, self.stayActiveMs)
```

Comment:
If the value is None for any of the fields, we have to use default value that is not zero, but rather the invalid value.

Reply:
https://github.com/project-chip/connectedhomeip/issues/33732, immediate follow-up with ticket this week here.